src/RosterContoller.cpp

```diff
@@ -26,12 +24,12 @@ void RosterController::handleRosterReceived(Swift::ErrorPayload::ref error)
 	}
 	else
 	{
-		std::cout << "handleRosterReceived!!!" << std::endl;
+		//std::cout << "handleRosterReceived!!!" << std::endl;
 		Swift::XMPPRoster* roster = client_->getRoster();
 		std::vector<Swift::XMPPRosterItem> rosterItems = roster->getItems();
 
 		std::vector<Swift::XMPPRosterItem>::iterator it;
-		std::cout << "size: " << rosterItems.size() << std::endl;
+		//std::cout << "size: " << rosterItems.size() << std::endl;
```

Comment:
Later we work on the status for each roster entry. Than we need it here again. Just wait for the Roster to be finished and then remove it.